analytics_data_api/v0/tests/views/test_learners.py

```diff
@@ -87,6 +102,7 @@ def tearDown(self):
         """Remove the index after every test."""
         super(LearnerAPITestMixin, self).tearDown()
         self._es.indices.delete(index=settings.ELASTICSEARCH_LEARNERS_INDEX)
+        self._es.indices.delete(index=settings.ELASTICSEARCH_LEARNERS_UPDATE_INDEX)
```

Comment:
I know it's not directly related to this PR, but would you mind refactoring this cleanup to use [`self.addCleanup`](https://docs.python.org/2/library/unittest.html#unittest.TestCase.addCleanup)? I recently learned that it's preferred over `self.tearDown`, since every statement in tearDown may not run if one of them fails.